docs/reference/migration/migrate_8_0/java_time.asciidoc

```diff
@@ -0,0 +1,20 @@
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+//tag::notable-breaking-changes[]
+
+// end::notable-breaking-changes[]
+
+[discrete]
+[[breaking_80_java_time_changes]]
+=== Replacing Joda-Time with java time
+
+All indices created in {es} 7 are already using java-time so nothing needs to be done.
+However indices created in {es} 6 with joda-time patterns need to be reindexed with
+an updated mappings
+
+[discrete]
+==== Migration guide
+For a detailed migration guide, see <<migrate-to-java-time>>.
+
+include::migrate_to_java_time.asciidoc[]
```

Comment:
Rewords the section and updates it to use the standard breaking changes format.
Because breaking changes are re-used in our Stack Guides, we need to use
the external link syntax instead of internal xrefs.

```suggestion
//tag::notable-breaking-changes[]
.Java-time date formats replace joda-time formats
[%collapsible]
====
*Details* +
In 7.0, {es} switched from joda time to java time for date-related parsing,
formatting, and calculations. Indices created in 7.0 and later versions are
already required to use mappings with java-time date formats. However, 
earlier indices using joda-time formats must be reindexed to use
mappings with java-time formats.

*Impact* +
For a detailed migration guide, see {ref}/migrate-to-java-time.html.
====

// end::notable-breaking-changes[]
```